### PR TITLE
completed css-pseudo-classes

### PR DESCRIPTION
**Pull request by Jc-Leon**
Screen Shot 2021-06-23 at 3 05 16 PM

### CODE REVIEW

File: css-pseudo-classes/style.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  color: white;
+  cursor: pointer;
+}
+th {
+  background-color: #000000;
+  color: white;
+ }
+ tr.one {
+  background-color:#D2D2D2;
+ }
```
Comment (codingcodymiller) [inline css-pseudo-classes/style.css:18]: Try to do this using the `nth-child()` pseudo-class

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice fix! Now that the class is no longer necessary, you should remove it from the HTML elements you applied it to.

**Comment by Jc-Leon** [comment]
thanks for keeping me on my toes  😅

**Review by codingcodymiller — APPROVED** [review]
LOL you got it!! I will always strive to keep you on your toes 😎

**Pull request merged**